templates/datasets/viewer.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
```

Comment:
Just wanted naming of templates to be consistent. We have *delete*, *editor*, and *view*. Perhaps I should rename *editor* instead. :confused: